node_scripts/setup_node.sh

```diff
@@ -39,7 +39,6 @@ fi
 # Unzip resource files and set permissions
 apt-get -y install unzip
 chmod 777 $AZ_BATCH_TASK_WORKING_DIR/docker_main.sh
-chmod -R 777 $AZ_BATCH_TASK_WORKING_DIR/custom-scripts
```

Comment:
wait isn't that needed to run the custom scripts?

Reply:
No - we chmod each script on the node before we run it, (here)[https://github.com/jafreck/aztk/blob/bug/remove-chmod-error/node_scripts/install/scripts.py#L47]. This was just causing an unnecessary error if the custom-scripts directory didn't exist.